graphql/documents/data/config.graphql

```diff
@@ -32,9 +32,6 @@ fragment ConfigGeneralData on ConfigGeneralResult {
   galleryExtensions
   excludes
   imageExcludes
-  scraperUserAgent
-  scraperCertCheck
-  scraperCDPPath
```

Comment:
For compatibility purposes, we want to avoid removing these right now. They should be marked as deprecated, but should still function correctly.

Reply:
I'm not too familiar with GraphQL and the usage in Stash. But aren't these fragments only used by the client side code? (in other words: used by the code generator for React). So removing these would then be fine I guess as long as the web interface doesn't use them? (which obviously isn't the case anymore)

Reply:
@WithoutPants any comment on this?

I did accidentally revert it as well, but as stated in aboves comment I'm not entirely sure if it was needed. This because I believe it's only used by the client side code (generator). So if that's the case different clients of the API aren't using this so removing it would be fine as the Stash web UI doesn't need it anymore.

Reply:
Oh, yes, you are correct. If it's client side it can be removed assuming it's unused.

Reply:
> If it's client side

Well, I'm not sure (either) if it is client side only :) But for now I left it in. But feel free to remove it if you think it is unnecessary.